### DISCUSSION TITLE
Lottie

### DISCUSSION THREAD

**Discussion by heshant3** [heshant3/react-lottie#5] [Q&A] [answered]
Do you know how to install Lottie

**Comment by Tharindukt3** [comment, marked as answer]
Check this one:
https://www.npmjs.com/package/lottie-react

**Reply by gihanjayakuru** [reply]
useful

**Comment by gihanjayakuru** [comment]
(https://www.hypermatic.com/tutorials/how-to-add-lottie-animations-to-figma-banner-exports-using-bannerify/)
